### PR TITLE
Improved query docs

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
- Adds details about return value of the querier endpoint
- Changed heading for `/api/echo` to better reflect what it does. (don't want to get mixed up with k8s /readiness handlers)

### CODE REVIEW

File: docs/tempo/website/api_docs/_index.md [hunk 1]
```diff
@@ -121,7 +121,11 @@ Parameters:
 Note that this API is not meant to be used directly unless for debugging the sharding functionality of the query 
 frontend.
 
-### Query Path Readiness Check
+Returns:
+By default this endpoint returns [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto/trace/v1) JSON,
+but if it can also send OpenTelemetry proto if `Accept: application/protobuf` is passed.
```
[the remark this answers is not in the document]
Comment (joe-elliott) [reply docs/tempo/website/api_docs/_index.md:126]: https://developers.google.com/protocol-buffers

**Review by kvrhdn — CHANGES_REQUESTED** [review]
Good call to rename the echo endpoint, readiness is confusing :+1:

You'll also have to update the table at the top of this page.

File: docs/tempo/website/api_docs/_index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ For the sake of clarity, in this document we have grouped API endpoints by servi
 | [Pprof](#pprof) | _All services_ |  HTTP | `GET /debug/pprof` |
 | [Ingest traces](#ingest) | Distributor |  - | See section for details |
 | [Querying traces](#query) | Query-frontend |  HTTP | `GET /api/traces/<traceID>` |
-| [Query Path Readiness Check](#query-path-readiness-check) | Query-frontend |  HTTP | `GET /api/echo` |
+| [Query Echo Endpoint](#query-path-readiness-check) | Query-frontend |  HTTP | `GET /api/echo` |
```
Comment (kvrhdn) [inline docs/tempo/website/api_docs/_index.md:24]:
```suggestion
| [Query Echo Endpoint](#query-echo-endpoint) | Query-frontend |  HTTP | `GET /api/echo` |
```

**Review by kvrhdn — APPROVED** [review]
Besides the section link, LGTM 🙂

**Review by annanay25 — APPROVED** [review]

**Pull request merged**